src/main/java/com/carrotcreative/cream/loaders/single/SingleLoader.java

```diff
@@ -118,7 +118,7 @@ public void always() {
      * This can be overwritten in a subclass if you feel
      * 3 isn't the right number
      */
-    protected int mWriteAttempts = 3;
+    private int mWriteAttempts = 3;
```

Comment:
The comment above actually explains that this can be overridden in a subclass, with private this is not possible!